File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/cacheRequest/api/CacheRequestImpl.java
package org.ngbp.jsonrpc4jtestharness.rsp.cacheRequest.api;

import org.ngbp.jsonrpc4jtestharness.rsp.cacheRequest.model.QueryCacheUsage;
import org.ngbp.jsonrpc4jtestharness.rsp.mapper.JsonRpcResponse;

public class CacheRequestImpl implements CacheRequest {
    @Override
    public JsonRpcResponse<CacheRequest> cacheRequest() {
        return null;
    }

    @Override
    public JsonRpcResponse<CacheRequest> CacheRequestDASH() {
        return null;
    }

    @Override
    public JsonRpcResponse<QueryCacheUsage> queryCacheUsage() {
        return null;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/cacheRequest/CacheRequestImpl.kt
package org.ngbp.jsonrpc4jtestharness.rpc.cacheRequest

import org.ngbp.jsonrpc4jtestharness.rpc.cacheRequest.model.CacheRequest
import org.ngbp.jsonrpc4jtestharness.rpc.cacheRequest.model.QueryCacheUsage

class CacheRequestImpl : ICacheRequest {
    override fun cacheRequest(): CacheRequest {
        return CacheRequest()
    }

    override fun CacheRequestDASH(): CacheRequest {
        return CacheRequest()
    }

    override fun queryCacheUsage(): QueryCacheUsage {
        return QueryCacheUsage()
    }
}